import-scripts/oncokb-annotator.sh

```diff
@@ -1,174 +1,298 @@
+            echo "Failed to annotate fusion file, exiting..."
+            ONCOKB_ANNOTATION_SUCCESS=0
+        fi
+    fi
+
+    # Annotating CNA
+    if [ $ONCOKB_ANNOTATION_SUCCESS -eq 1 ] ; then
+        echo $(date)
+        echo "Beginning CNA annotation..."
+        $PYTHON_BINARY $CNA_ANNOTATOR_SCRIPT -b $ONCOKB_TOKEN -i $STAGING_CNA_FILE -o $ONCOKB_CNA_FILE -c $STAGING_SAMPLE_FILE -u $ONCOKB_URL
+        if [ $? -ne 0 ] ; then
+            echo "Failed to annotate CNA file, exiting..."
+            ONCOKB_ANNOTATION_SUCCESS=0
+        fi
+    fi
+
+    # Annotating Clinical
+    if [ $ONCOKB_ANNOTATION_SUCCESS -eq 1 ] ; then
+        echo $(date)
+        echo "Beginning clinical annotation..."
+        $PYTHON_BINARY $CLINICAL_ANNOTATOR_SCRIPT -i $STAGING_SAMPLE_FILE -o $ONCOKB_SAMPLE_FILE -a $ONCOKB_MAF_FILE,$ONCOKB_FUSION_FILE,$ONCOKB_CNA_FILE
+        if [ $? -ne 0 ] ; then
+            echo "Failed to annotate clinical file, exiting..."
+            ONCOKB_ANNOTATION_SUCCESS=0
+        fi
+    fi
+
+    # Generating Clinical PDF
+    if [ $ONCOKB_ANNOTATION_SUCCESS -eq 1 ] ; then
+        echo $(date)
+        echo "Beginning clinical PDF generation..."
+        $PYTHON_BINARY $PDF_SCRIPT -i $ONCOKB_SAMPLE_FILE -o $ONCOKB_PDF_FILE -n $ONCOKB_PDF_NUMBER
+        if [ $? -ne 0 ] ; then
+            echo "Failed to generate clinical pdf, exiting..."
+            ONCOKB_ANNOTATION_SUCCESS=0
+        fi
+    fi
+
+    # Annotating Somatic Clinical
+    if [ $ONCOKB_ANNOTATION_SUCCESS -eq 1 ] ; then
+        echo $(date)
+        echo "Beginning somatic clinical annotation..."
+        # Generate somatic-only MAF by excluding lines including 'GERMLINE'
+        awk -F'\t' '$26 != "GERMLINE"' $ONCOKB_MAF_FILE > $ONCOKB_SOMATIC_MAF_FILE
+        $PYTHON_BINARY $CLINICAL_ANNOTATOR_SCRIPT -i $STAGING_SAMPLE_FILE -o $ONCOKB_SOMATIC_SAMPLE_FILE -a $ONCOKB_SOMATIC_MAF_FILE,$ONCOKB_FUSION_FILE,$ONCOKB_CNA_FILE
+        if [ $? -ne 0 ] ; then
+            echo "Failed to annotate somatic clinical file, exiting..."
+            ONCOKB_ANNOTATION_SUCCESS=0
+        fi
+    fi
+
+    # Generating Somatic Clinical PDF
+    if [ $ONCOKB_ANNOTATION_SUCCESS -eq 1 ] ; then
+        echo $(date)
+        echo "Beginning somatic clinical PDF generation..."
+        $PYTHON_BINARY $PDF_SCRIPT -i $ONCOKB_SOMATIC_SAMPLE_FILE -o $ONCOKB_SOMATIC_PDF_FILE -n $ONCOKB_PDF_NUMBER
+        if [ $? -ne 0 ] ; then
+            echo "Failed to generate somatic clinical pdf, exiting..."
+            ONCOKB_ANNOTATION_SUCCESS=0
+        fi
+    fi
+
+    # Generating README
+    if [ $ONCOKB_ANNOTATION_SUCCESS -eq 1 ] ; then
+        echo $(date)
+        echo "Beginning README generation..."
+        $PYTHON_BINARY $README_SCRIPT -o $ONCOKB_README_FILE -u $ONCOKB_URL
+        if [ $? -ne 0 ] ; then
+            echo "Failed to generate README, exiting..."
+            ONCOKB_ANNOTATION_SUCCESS=0
+        fi
+    fi
+
+    # Only commit if all steps succeeded
```

Comment:
Only rsync if all steps succeeded (instead of commit?)

Reply:
right .. there is no modification of the repository here. We upload the annotated oncokb files to ramen as a community download.